src/Makevars.in

```diff
@@ -26,8 +26,8 @@ BBG_RPATH   = '$$ORIGIN/../blp'
 
 ## set include and linker options
 ## Bbg API files are assummed to be in the standard search path
-PKG_CPPFLAGS = -I../inst/include/ -I.
-PKG_LIBS     = -l$(BBG_LIB) -L../inst/blp -Wl,-rpath,$(BBG_RPATH)
+PKG_CPPFLAGS = -I../inst/include/ -I. @badsystem@
+PKG_LIBS     = $(BBG_LIB) -L../inst/blp -Wl,-rpath,$(BBG_RPATH)
 
 all: $(SHLIB)
-	@if command -v install_name_tool; then echo "fixing"; install_name_tool -add_rpath @loader_path/../blp Rblpapi.so; fi
+	@if command -v install_name_tool@badsystem@; then echo "fixing"; install_name_tool -add_rpath @loader_path/../blp Rblpapi.so; fi
```

Comment:
Why `@badsystem@` here? I don't follow

Reply:
I apologize for the sneakyness :grinning: but it safes another variable.  Basically in the 'false' case it appends the `-DNoBlpHere` creating a `type -p` or `which` tests implemented more portably via `command -v` for a binary that does not exist so we skip the call of `install_name_tool` ... which would otherwise fail and break as there is no Rblpapi.so.

As the CI runs show, it works in the two 'present' cases; it also works when I tested a bad system but just moving lines 38 to 43 to the top of `configure`.  I recommend you do that to even on a system that has blp support just to run through the 'false' case.

Reply:
Clever, I like it. Can we maybe add an explanatory comment? I'm unlikely to remember this years from now

Reply:
Deal. Will commit a line or two.

Reply:
I think medium-term we could make src/Makevars.in clearer by having configure assign an overall variable "HAVE_BLP" in the src/Makevars and then if/else branch based on that variable.

On the other hand what we have now _works_ as I gave it chase earlier so it would take some fresh effort to rewrite ...